packages/atomic/src/components/atomic-folded-result-list/atomic-folded-result-list.tsx

```diff
@@ -0,0 +1,427 @@
+ *
+ * @part result-list - The element containing every result of a result list
+ * @part result-list-grid-clickable - The clickable result when results are displayed as a grid
+ * @part result-table - The element of the result table containing a heading and a body
+ * @part result-table-heading - The element containing the row of cells in the result table's heading
+ * @part result-table-heading-row - The element containing cells of the result table's heading
+ * @part result-table-heading-cell - The element representing a cell of the result table's heading
+ * @part result-table-body - The element containing the rows of the result table's body
+ * @part result-table-row - The element containing the cells of a row in the result table's body
+ * @part result-table-cell - The element representing a cell of the result table's body
+ */
+@Component({
+  tag: 'atomic-folded-result-list',
+  styleUrl: 'atomic-folded-result-list.pcss',
+  shadow: true,
+})
+export class AtomicFoldedResultList implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
+  private resultList!: FoldedResultList;
+  public resultsPerPage!: ResultsPerPage;
+  private resultTemplatesManager!: ResultTemplatesManager<TemplateContent>;
+
+  @Element() private host!: HTMLDivElement;
+
+  @BindStateToController('resultList')
+  @State()
+  private resultListState!: FoldedResultListState;
+
+  @BindStateToController('resultsPerPage')
+  @State()
+  private resultsPerPageState!: ResultsPerPageState;
+
+  @State() public error!: Error;
+  @State() private templateHasError = false;
+
+  /**
+   * TODO:
+   */
+  @Prop({reflect: true}) public foldingCollection?: string;
+  /**
+   * TODO:
+   */
+  @Prop({reflect: true}) public foldingParent?: string;
+  /**
+   * TODO:
+   */
+  @Prop({reflect: true}) public foldingChild?: string;
+
+  /**
+   * TODO: KIT-452 Infinite scroll feature
+   * Whether to automatically retrieve an additional page of results and append it to the
+   * current results when the user scrolls down to the bottom of element
+   */
+  private enableInfiniteScroll = false;
+  /**
+   * A list of non-default fields to include in the query results, separated by commas.
+   * The default fields sent in a request are: 'date', 'author', 'source', 'language', 'filetype', 'parents', ‘urihash’, ‘objecttype’, ‘collection’, ‘permanentid’ 'ec_price', 'ec_name', 'ec_description', 'ec_brand', 'ec_category', 'ec_item_group_id', 'ec_shortdesc', 'ec_thumbnails', 'ec_images', 'ec_promo_price', 'ec_in_stock', 'ec_cogs', and 'ec_rating'.
+   */
+  @Prop({reflect: true}) public fieldsToInclude = '';
+  /**
+   * The desired layout to use when displaying results. Layouts affect how many results to display per row and how visually distinct they are from each other.
+   */
+  @Prop({reflect: true}) display: ResultDisplayLayout = 'list';
```

Comment:
Gonna have to check with Johnny, but I imagine table and grid won't be supported, so you'll probably be able to remove this prop.